src/bentoml/_internal/runner/runner_handle/remote.py

```diff
@@ -77,9 +77,9 @@ def runner_timeout(self) -> int:
         "return the configured timeout for this runner."
         runner_cfg = BentoMLContainer.runners_config.get()
         if self._runner.name in runner_cfg:
-            return runner_cfg[self._runner.name]["timeout"]
+            return runner_cfg[self._runner.name].get("traffic", {})["timeout"]
```

Comment:
This will raise KeyError if the default is an empty dict?

Reply:
traffic dict is optional but the timeout value isn't, so it is guaranteed to have a `timeout` value in the dict.